src/main/java/duke/command/DukeCommands.java

```diff
@@ -0,0 +1,51 @@
+package duke.command;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+
+/**
+ * The DukeCommands enum encapsulates the logic for commands.
+ */
+public enum DukeCommands {
```

Comment:
Should this be singular instead? When instantiating a command it sounds a bit weird to say `DukeCommands command`